### ISSUE TITLE
Make it Better

### ISSUE THREAD

**Issue by Kirbdawg** [Kreolix/CovenantMissionHelper#31]
Although your addon is a godsent, I still use the WeakAuras version too.  I use it because it tells me how close i am.   First I fill all the slots with 3 nightblades and 2 soul casters(these are the Venthyr troops, melee and ranged).   WA tells me how close I am... and whether I need a companion with a high ilevel, or if I can use a low ilevel one.   This way saves me a lot of scrolling up and down.

A button to load the default troops would be awesome... I'd never have to scroll down to see them again.

A counter that provides the chance to win like WA and another addon does, helps more than you know.

![Perfect](https://user-images.githubusercontent.com/41462842/107569594-1b36f480-6bae-11eb-8b3c-c0be88865b45.jpg)

**Comment by chadmccune** [comment]
I agree, awesome addon.  Would love a feature that just optimized and executed each mission for me automatically...I don't really care for the missions, so something to knock them out as fast as possible would be amazing...

Click Table -> Optimize -> (wait on addon to do all the permutations) -> next next next next next ... until done :)

**Comment by Gaboros** [comment]
Sent a PR where the optimize button tries to fill up the board with troops.
